README.md

```diff
@@ -259,6 +261,24 @@ Reference projects which are built around Zero configuration:
 
 </details>
 
+### 1.8 Respect POSIX signals
+
+✅ **Do:**
+Ensure your program respects [POSIX signals](http://man7.org/linux/man-pages/man7/signal.7.html) to allow it proper interaction with users or other programs.
+
+❌ **Otherwise:**
+Your program will not play well with other programs and introduce unexpected behavior.
+
+<details>
+  <summary>➡️ <b>Details</b></summary>
+
+Especially for CLI applications, it is common to interact with user input and improperly managing keyboard events
+may result in unresponsive program to the `SIGINT` signal interrupt which is commonly used by users when they hit the `CTRL+C` keys.
```

Comment:
change "may result in unresponsive program to the `SIGINT` signal interrupt which is" to "may result in your app failing to respond to `SIGINT` interrupts, "

Reply:
```suggestion
may result in your app failing to respond to SIGINT interrupts, commonly used by users when they hit the `CTRL+C` keys.
```

Reply:
Changed it, thanks @davesag you're the best!